hapi-fhir-jpaserver-hfql/src/main/java/ca/uhn/fhir/jpa/fql/jdbc/JdbcDriver.java

```diff
@@ -0,0 +1,117 @@
+/*-
+ * #%L
+ * HAPI FHIR JPA Server - HFQL Driver
+ * %%
+ * Copyright (C) 2014 - 2023 Smile CDR, Inc.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package ca.uhn.fhir.jpa.fql.jdbc;
+
+import java.io.PrintStream;
+import java.sql.*;
+import java.util.Properties;
+import java.util.logging.Logger;
+
+/**
+ * This is the JDBC driver class for the HFQL driver. It is intended to be
+ * imported into a JDBC-compliant database tool, and implements the basic
+ * functionality required to introspect the "database" and execute queries.
+ * <p>
+ * Connections returned by this driver are only semi-stateful. In a normal
+ * JDBC driver, each connection represents an open and persistent TCP
+ * connection to the server with shared state between the client and the
+ * server, but in this driver we keep most of the state in the client. When
+ * a query is executed it is translated into a FHIR search (with further
+ * processing on the search results happening in
+ * {@link ca.uhn.fhir.jpa.fql.executor.HfqlExecutor}).
+ */
+public class JdbcDriver implements Driver {
+	private static final JdbcDriver INSTANCE = new JdbcDriver();
+	public static final String URL_PREFIX = "jdbc:hapifhir:hfql:";
```

Comment:
suggestion: We can just use a string-prefix:
```suggestion
	public static final String URL_PREFIX = "jdbc:hapifhirql:";
```